lesson5_task3.py

```diff
@@ -0,0 +1,21 @@
+"""
+Words combination
+
+Create a program that reads an input string and then creates and prints 5 random strings from characters of the input
+string. For example, the program obtained the word ‘hello’, so it should print 5 random strings(words)
+that combine characters 'h', 'e', 'l', 'l', 'o' -> 'hlelo', 'olelh', 'loleh' …
+
+Tips: Use random module to get random char from string)
+"""
+import random
```

Comment:
Nice code style, you could also try to replace:
`for example in range(5)` with `for _ in range(5)` if you don't need the value which you iterate over.